src/ssh/ssh_process.cpp

```diff
@@ -32,6 +32,28 @@
 namespace mp = multipass;
 namespace mpl = multipass::logging;
 
+constexpr auto category = "ssh process";
+
+consteval std::string_view extract_filename(std::string_view path)
+{
+    auto pos = path.find_last_of("/\\");
+    return (pos == std::string_view::npos) ? path : path.substr(pos + 1);
+}
+
+void log_trace(const std::string_view message,
```

Comment:
Hi @prachimaskar, this doesn't appear to be used yet. Did you forget to replace the log calls?